client/cypress/e2e/pages/userprofile.cy.js

```diff
@@ -280,6 +280,7 @@ describe("Check user profile", () => {
     const rankingClass = "Schirme EN A+B mit einer Streckung <5,2";
     cy.get("#addGliderModal").within(() => {
       cy.get("#brand-select").select(brand).should("have.value", brand);
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
Brauchen wir das wait hier wirklich? Durch das should im Anschluss an das select sollte doch hier gewartet werden, bis der richtige Wert selektiert wurde

Reply:
Probier es aus und schau zu wie der Test in der ci failed🤪